examples/boutcore/blob2d.py

```diff
@@ -154,7 +145,145 @@ def rhs(self, time):
         self.omega.ddt(ddt_omega)
 
 
-# Create an instance
-blob2d = Blob2D()
-# Start the simulation
-blob2d.solve()
+# Ensure the blob folder exists
+def ensure_blob():
+    if not os.path.isdir("blob"):
+        print("Setting up folder blob for simulation ...")
+        os.mkdir("blob")
+    if not os.path.exists("blob/BOUT.inp"):
+        with open("blob/BOUT.inp", "w") as f:
+            f.write(
```

Comment:
I guess you're writing this as a string, rather than from a `BoutOptions` is to preserve the comments?

Reply:
Mostly because a string is easy to store in a source file. If you know how to create the code for generating a BoutOptions, I wouldn't mind that either. But having a string is very easy to copy-paste between the two files and keep them in sync.

Reply:
It's mostly like a dict, though I think constructing one from a dict is less than ideal at the mo.